src/transformers/models/bart/modeling_bart.py

```diff
@@ -1318,6 +1318,9 @@ def forward(
         return_dict = return_dict if return_dict is not None else self.config.use_return_dict
 
         if labels is not None:
+            if use_cache:
+                warnings.warn("The `use_cache` argument is changed to `False` since `labels` is provided.")
```

Comment:
Sorry can we use `logger.warn(...)` here instead. We only use `warnings.warn(...)` for Deprecation warnings I think. cc @sgugger @LysandreJik  no?

Reply:
Indeed.

Reply:
Changed it. (use `logger.warning` though)